apps/marketplace/src/app/app-routing.module.ts

```diff
@@ -30,6 +30,10 @@ const routes: Routes = [
   {
     path: 'faq',
     loadChildren: () => import('./faq/faq.module').then(m => m.FAQModule)
+  },
+  {
+    path: 'player',
```

Comment:
This should be in the player app not the marketplace.
Also add the tokenId as params.